rules/base.js

```diff
@@ -46,7 +46,6 @@ module.exports = {
 		'accessor-pairs': 2, // enforce getter and setter pairs in objects
 		'array-callback-return': 0, // enforce `return` statements in callbacks of array methods
 		'block-scoped-var': 0, // enforce the use of variables within the scope they are defined
-		'class-methods-use-this': 1, // enforce that class methods utilize `this`
```

Comment:
Let's keep the rule in but set it to 0. I like maintaining a config of every single current rule in the system.